src/compas_fab/robots/robot.py

```diff
@@ -1072,28 +1047,51 @@ def inverse_kinematics_deprecated(self, frame_WCF, start_configuration=None,
 
         return configuration.scaled(self.scale_factor)
 
+    def inverse_kinematics_deprecated(self, frame_WCF, start_configuration=None,
```

Comment:
I reversed the call hierarchy between `xyz` and their `xyz_deprecated` method, now deprecated methods call the non-deprecated versions, so that we can easily delete them on a future release. It also makes the non-deprecated code simpler because it doesn't need to know much about what's in the `options`.